### DISCUSSION TITLE
Formiddable CVE

### DISCUSSION THREAD

**Discussion by vuthikxkol** [serverless/serverless#12440] [Q&A] [answered]
I noticed that a dependency of serverless, formidable, has a critical CVE. Serverless@3.38 uses formidable@2.1.1 which is vulnerable according to https://github.com/advisories/GHSA-8cp3-66vr-3r4c. Fix version is formidable@3.2.4. Since overriding would mean moving across major versions. I'm worried about breaking changes. Does anyone know if this is safe to do so or if there is any other workaround for this CVE?

**Comment by czubocha** [comment, marked as answer]
The CVE referenced for formidable ([GHSA-8cp3-66vr-3r4c](https://github.com/advisories/GHSA-8cp3-66vr-3r4c)) has actually been withdrawn, meaning it’s no longer considered a valid vulnerability.

As a result, upgrading to formidable@3.2.4 specifically for this CVE isn’t necessary. You should be able to continue using the current version without security concerns related to this advisory.

For more information, you can check [the advisory status directly on GitHub](https://github.com/advisories/GHSA-8cp3-66vr-3r4c).
